tests/test_plugins_apps_av_mcafee.py

```diff
@@ -0,0 +1,16 @@
+from dissect.target import Target
+from dissect.target.plugins.apps.av.mcafee import McAfeeLogRecord
+
+from ._utils import absolute_path
+
+
+def test_mcafee_plugin_log():
+    for record in Target.open(absolute_path("data/apps/mcafee/mcafee.tar.gz")).mcafee.logs():
```

Comment:
```suggestion
    for record in Target.open(absolute_path("data/apps/mcafee/mcafee.tar.gz")).mcafee.logs():
```
Can you move the test file to the `data/apps/av/mcafee` directory?

Reply:
Moved to av-dir.